src/Microsoft.TestPlatform.PlatformAbstractions/Interfaces/System/IProcessHelper.cs

```diff
@@ -103,5 +103,12 @@ public interface IProcessHelper
         /// </summary>
         /// <param name="process">Reference to process</param>
         void WaitForProcessExit(object process);
+
+        /// <summary>
+        /// Gets the process handle for given process Id.
+        /// </summary>
+        /// <param name="processId">process id</param>
+        /// <returns>Process Handle</returns>
+        IntPtr GetProcessHandleById(int processId);
```

Comment:
Nit: Method name can be simply `GetProcessHandle`

Reply:
Updated the name to `GetProcessHandle`